### ISSUE TITLE
'should.and' failing with 'karma@0.13'

### ISSUE THREAD

**Issue by AlexKVal** [karma-runner/karma#1498]
This spec-code
```js
var methodTest = sinon.stub();

methodTest('y');

methodTest.should.have.been.calledOnce
    .and.calledWith('y');
```
causes
```
✖ validates each validation
      Chrome 43.0.2357 (Mac OS X 10.9.5)
    TypeError: Cannot read property 'and' of undefined
```
with `karma@0.13` version.

--
I've created simple project for bug repro:
https://github.com/AlexKVal/karma-test

`master` branch uses `karma@0.13` and causes test fail
karma13 https://travis-ci.org/AlexKVal/karma-test/builds/71240835
<img width="748" alt="screen shot 2015-07-16 at 3 12 23 pm" src="https://cloud.githubusercontent.com/assets/847572/8722468/af4240de-2bcd-11e5-933c-0a8e855c5586.png">

https://github.com/AlexKVal/karma-test/tree/karma12 branch uses `karma@0.12` version and works fine
karma12 https://travis-ci.org/AlexKVal/karma-test/builds/71240954
<img width="418" alt="screen shot 2015-07-16 at 3 05 59 pm" src="https://cloud.githubusercontent.com/assets/847572/8722475/c24d7fcc-2bcd-11e5-938a-0230fb55a677.png">

**Comment by Dignifiedquire** [comment]
Thanks for the report, will try to take look as soon as I can

**Comment by Dignifiedquire** [comment]
It looks like the underlying issue is the same as #1499. There is a conflict between the `karma-chai` and the `karma-sinon-chai` plugin. They both include a version of `chai`, and in `0.12` the addition from `karma-sinon-chai` simply won, but for some reason this broken now. You can work around it for now by removing `karma-chai`, which isn't needed anyway.

**Comment by AlexKVal** [comment]
Thank you @Dignifiedquire for the hint about `karma-chai`. :cherries: 
Indeed it helps with tests. :sparkle: 

Now all is left to migrate `react-bootstrap` to `karma@0.13` is https://github.com/karma-runner/karma/issues/1497, which as I can see is also caused by the new file loading mechanic.

**Comment by Dignifiedquire** [comment]
Fix was released in `0.13.1`

**Comment by AlexKVal** [comment]
Thank you :tada: :cherries:
